### ISSUE TITLE
High level overview of all tasks

### ISSUE THREAD

**Issue by kiview** [kiview/winkie#1]
The following stuff needs to be done. We should split thi

- [x] read in DLC result files
- [ ] cleanup/pre-process result files
  - [ ]  check interpolation quality of DLC, or use our own interpolation approach
  - [ ]   write out resulting data into new files
- [ ] data preparation  
  - [x]   relative coordinates
  - [x]   rotate the pigeon for normalization
  - [ ]   normalization of scale necessary? (check differences in size of pigeons and if needed)
- [x] labelling
  - [x] convert extracted frames to labelled input file (GitHub + GoogleDoc?)
  - [x]   make labelling easier?
  - [x]   apply labels on training data

- [ ] create AI model
  - [x]   e.g use our Keras LSTM model
  - [x]   easy way to tune and specify hyperparameters
  - [x] prepare training and test data
  - [ ] potentially add padding to training data
  - [x] train model
  - [x] evalute results

- [ ] _Side work: Generate more training data, e.g. by adding noise_
